blockedModeSetup.js

```diff
@@ -51,6 +51,7 @@ function timer(){
 				if (timeLeft < 0) {
 					clearInterval(timerInterval);
 					document.getElementById("timer").innerHTML = "UNBLOCKED!";
+					//turnFilteringOff();
```

Comment:
to chyba trzeba zlikwidować?  Czy sugestią miało być

Reply:
Sugestią. ^^